arch/arm/dts/imx8mp-var-dart-sapphire.dts

```diff
@@ -110,7 +110,25 @@
 	status = "okay";
 };
 
+&gpio1 {
```

Comment:
Do we need still need these changes?

Reply:
yes we are using a load switch to enable power for sd card

Reply:
SD card HW has been deprecated. But let's keep it for now.